plugin/completion.py

```diff
@@ -166,7 +165,7 @@ def do_request(self, completion_list: sublime.CompletionList, prefix: str, locat
             return
 
         self.manager.documents.purge_changes(self.view)
-        document_position = get_document_position(self.view, locations[0])
+        document_position = text_document_position_params(self.view, locations[0])
         if document_position:
```

Comment:
I removed the need for this `if` check. text_document_position_params always returns a dict, wereas get_document_position returned an optional dict.